wm_rhasspy_ros_bridge/wm_rhasspy_ctrl wm_rhasspy_ctrl
---
bool sucess